common.sh

```diff
@@ -236,8 +236,28 @@ fi
 # Check CentOS version
 VER=$(awk -F= '/^VERSION_ID=/ { print $2 }' /etc/os-release | tr -d '"' | cut -f1 -d'.')
 if [[ ${VER} -ne 7 ]] && [[ ${VER} -ne 8 ]]; then
-  echo "Required CentOS 7 / RHEL 7 / RHEL 8"
-  exit 1
+  if [[ ${VER} -eq 7 ]] && [[ "${ALLOW_CENTOS7}" -ne "yes" ]]; then
```

Comment:
Does this work? The if statement above on L238 is `if [[ ${VER} -ne 7 ]] && [[ ${VER} -ne 8 ]]`, so I think you won't get to this one on CentOS 7

Reply:
sorry .. obviously did this way too fast